src/js/background/context-menu.js

```diff
@@ -55,13 +55,13 @@ const staticMenuData = {
       "pageInputIconUseExistingAliasFromTheSite_mask"
     ),
     visible: true,
-    contexts: ["all"],
+    contexts: ["editable"],
```

Comment:
suggestion (non-blocking): I would recommend leaving this one as is. Having a way for users to upgrade may be useful? This may also seem like too upgrade-focused. LMK your thoughts.  

```suggestion
    contexts: ["all"],
```

Reply:
And here too, this is actually `useExistingAliasFromWebsite` rather than `upgradeToPremium` :)